modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/AndesConfigurationManager.java

```diff
@@ -60,14 +81,16 @@
     private static final String LIST_TYPE = "LIST_";
 
     /**
+     * Namespace and attribute used by secure vault to identify encrypted properties.
+     */
+    private static final QName SECURE_VAULT_QNAME = new QName("http://org.wso2.securevault/configuration","secretAlias");
+
+    /**
      * Common Error states
      */
     private static final String GENERIC_CONFIGURATION_PARSE_ERROR = "Error occurred when trying to parse " +
             "configuration value {0}.";
 
-    private static final String NO_CHILD_FOR_INDEX_IN_PROPERTY = "There was no child at the given index {0} for the " +
```

Comment:
Removing unused property. (Was used for the earlier mqtt authentication based on users in broker.xml.)